ntbs-service/wwwroot/source/Components/ValidateInput.ts

```diff
@@ -35,6 +36,7 @@ const ValidateInput = Vue.extend({
                         }
                     } else {
                         this.$el.classList.remove("nhsuk-form-group--error");
+                        this.$refs["errorField"].classList.add("hidden");
```

Comment:
This has potential to make small layout changes on validations that are not currently using the has-error tag helper. I think this change makes sense here, and the minor shuffles aren't _too_ bad. Just thought I would highlight it here.

Reply:
Good point, might need to semi prioritise that tech debt ticket!